ui/src/app/views/breadcrumb/component.ts

```diff
@@ -85,6 +85,9 @@ export class BreadcrumbComponent implements OnInit, OnDestroy {
       if (label === 'Param: Cohort Name') {
         label = child.snapshot.data['cohort'].name;
       }
+      if (label === 'Param: Concept Set Name') {
```

Comment:
Hmmm, we should maybe coordinate on this, because I was using the same param for the concept sets list page, but there was some funky stuff I had to do to prevent dup breadcrumbs... You can look at our PR for Concept Set Creation and list page for details

Reply:
Per offline discussion Ursa is checking in with @lstrano-broad on this. The mocks are not consistent right now.

For consistency with the rest of the app as currently implemented, I'd expect:

- List page: Workspaces > workspaceName > Concept Sets
- Details page: Workspaces > workspaceName > Concept Sets > conceptSetName